### ISSUE TITLE
Why my loss have no changes?

### ISSUE THREAD

**Issue by pixixiaonaogou** [balancap/SSD-Tensorflow#236]
I have train the pretrained_model on my own datasets by using this command 

DATASET_DIR=./tfrecords
TRAIN_DIR=./log/
CHECKPOINT_PATH=./checkpoints/vgg_16.ckpt
python train_ssd_network.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --dataset_split_name=train \
    --model_name=ssd_300_vgg \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --checkpoint_model_scope=vgg_16 \
    --checkpoint_exclude_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --trainable_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --save_summaries_secs=60 \
    --save_interval_secs=600 \
    --weight_decay=0.0005 \
    --optimizer=adam \
    --learning_rate=0.001 \
    --learning_rate_decay_factor=0.94 \
    --batch_size=32
Some problems in this process is solved by me .
But the loss have no changes in the process of training
So i want to ask some help for this question

**Comment by pixixiaonaogou** [comment]
And I provide some detail for my datasets
NUM_CLASSES = 1
and the loss is 1.3292 all the time .

**Comment by Gwwwydion** [comment]
@pixixiaonaogou you should make NUM_CLASSES = 2

**Comment by jhgfkdj** [comment]
Me too, and my total_loss always wave at 100 up and down and never devline , have you solved this problem?

**Comment by yyc9268** [comment]
I guess the NUM_CLASSES should be 2 because it includes background class. You can confirm that default NUM_CLASSES is set to 21 although PASCAL VOC includes only 20 classes.
